README.md

```diff
@@ -141,7 +141,7 @@ const ws = new WebSocket('ws://www.host.com/path', {
 ### Sending and receiving text data
 
 ```js
-import WebSocket from 'ws';
+import * as WebSocket from 'ws';
 
 const ws = new WebSocket('ws://www.host.com/path');
```

Comment:
This doesn't work.

![kuva](https://user-images.githubusercontent.com/5271115/129501824-3129ad32-645c-407e-bb32-cc34ba0228cd.png)

`const ws = new WebSocket.WebSocket('ws://www.host.com/path')`, but that one...